pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -271,7 +271,7 @@ void processArguments() throws Exception {
         protected Boolean retainKeyOrdering;
         @Parameter(names = "--batch-builder", description = "BatcherBuilder provides two types of batch construction methods, DEFAULT and KEY_BASED. The default value is: DEFAULT")
         protected String batchBuilder;
-        @Parameter(names = "--forward-source-message-property", description = "Forwarding input message's properties to output topic when processing")
+        @Parameter(names = "--forward-source-message-property", description = "Forwarding input message's properties to output topic when processing", arity = 1)
```

Comment:
can we update the doc to reflect the change, like `to disable it use --forward-source-message-property false`?
as user usually won't expect to provide parameter for a `flag` 
also please rebase on latest master, there's a fix on CI

Reply:
done